engine/graphics/src/test/wscript

```diff
@@ -12,7 +12,7 @@ def build(bld):
                          uselib_local = 'graphics_null graphics_transcoder_null',
                          target = name)
 
-    print "MAWE", platform_supports_feature(bld.env.PLATFORM, 'vulkan', {}), bld.env.PLATFORM 
+    print "MAWE", platform_supports_feature(bld.env.PLATFORM, 'vulkan', {}), bld.env.PLATFORM

```

Comment:
it's a debug print, it shouldn't be there :)

Reply:
True :) I can remove it